os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -79,6 +80,13 @@ static void ns_setstatusbits(u32 NewState)
 	device_mutex_unlock(RT_DEV_LOCK_FLASH);
 }
 
+static int ns_printf(const char *input)
+{
+	int ret;
+	ret = sedbg("%s \n", input);
```

Comment:
`Xdbg` functions have dependency on configuration like CONFIG_DEBUG_ERROR, CONFIG_DEBUG_XXX_ERROR.
I think you'd better to use printf instead of sedbg to print them always.

Reply:
printf is for application, SE layer is Kernel layer.
I think we should not use printf.

Reply:
What we print for kek is necessary regardless of debug config so that @jeongarmy asked you to change it to printf. For the same reason, we use some printf in kernel.

Reply:
As of now, we don't have any plan to add debug message more. If you add debug message additionally, (make current one as mandatory message), then we can choose proper way for each message. (But no plan to add another message)

Reply:
Understand, I have modify it to use printf.

Reply:
Thank you